pnpm-lock.yaml

```diff
@@ -35673,7 +35768,6 @@ packages:
     resolution: {integrity: sha512-eSRppjcPIatRIMC1U6UngP8XFcz8MQWGQdt1MTBQ7NaAmvXDfvNxbvWV3x2y6CdEUciCSsDHDQZbhYaB8QEo2g==}
     engines: {node: ^10 || ^12 || ^13.7 || ^14 || >=15.0.1}
     hasBin: true
```

Comment:
Seems like the 3.3.3 version should go away from deduplication - two versions now non-dev.

Reply:
I don't know how to force this other than `pnpm dedupe` but that will make other changes. Are these pnpm bugs? We
shouldn't have to do so much manual lockfile maintenance.

Reply:
@tylerbutler - You probably already know this, but I've noticed that `pnpm add <package>` or `pnpm install <package>` results in gratuitious pnpm churn.

If I revert the resulting lockfile and `pnpm i` it does better.

Reply:
@DLehenbauer Yeah that's my go-to, but in this case it only addresses some of the weirdness. The duplicate ts-jest is still there, for example.